roles/openshift_logging_elasticsearch/vars/openshift-enterprise.yml

```diff
@@ -1,5 +1,5 @@
 ---
 __openshift_logging_elasticsearch_image_prefix: "{{ openshift_logging_image_prefix | default('registry.access.redhat.com/openshift3/') }}"
-__openshift_logging_elasticsearch_image_version: "{{ openshift_logging_image_version | default ('v3.7') }}"
+__openshift_logging_elasticsearch_image_version: "{{ openshift_logging_image_version | default (openshift_version) }}"
```

Comment:
What's the difference between openshift_release and openshift_version?

Reply:
IIUC `openshift_version` is exact RPM/container version, e.g. `v3.9.0-0.16.0.0` and `openshift_release` is more generic. It seems some components don't have builds for every openshift build, so I'll replace it with `openshift_release` to make sure the latest minor version is used